packages/cfd/src/Containers/cfd-financial-stp-real-account-signup.tsx

```diff
@@ -117,14 +124,20 @@ const CFDFinancialStpRealAccountSignup = (props: TCFDFinancialStpRealAccountSign
         }
         return need_poi_for_bvi_labuan;
     };
-    const should_show_poa = !['pending', 'verified'].includes(authentication_status.document_status);
+
+    const shouldShowPOA = () => {
```

Comment:
We have a test file for this which is not updated and it is one of the reasons why codecov is failing.
Help to update the test file.